system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -218,6 +218,54 @@
       @include context-menu-button-hover;
     }
   }
+
+  &.top-sites-1rows {
+    @media (max-width: $break-point-widest - 1) {
+      .top-site-outer {
+        display: none;
+
+        &:nth-child(-n+6) {
```

Comment:
If we go with this approach, instead of `none` for all then "unhiding" the first 6, you could just hide not-the-first-6 with `.top-site-outer:nth-child(n+7) { display: none; }`

Reply:
I'll try this but for some reason I thought that would just not show any child.

Reply:
oh nice, it works. how did you trigger that edge case you screenshoted?